keylime/ima/ima.py

```diff
@@ -477,7 +498,54 @@ def read_allowlist(al_path=None, checksum="", gpg_sig_file=None, gpg_key_file=No
                 f"Checksum of allowlist does not match! Expected {checksum}, Calculated {calculated_checksum}"
             )
 
+    # Return artifacts as a JSON bundle, ready for sending
+    bundle = {
+        "ima_policy": base64.b64encode(alist_bytes).decode(),
+        "key": base64.b64encode(al_key).decode(),
+        "sig": base64.b64encode(al_sig).decode(),
+        "checksum": checksum,
```

Comment:
The checksum is never validated, right?

Reply:
The checksum value is also used during allowlist processing (which is now on the verifier side), hence why it's being sent along with the signing mateials. I suppose if we wanted we could also do a checksum check on the verifier side as well.